addons/web/static/tests/widgets/data_export_tests.js

```diff
@@ -100,16 +94,73 @@ QUnit.module('widgets', {
             viewOptions: {
                 hasActionMenus: true,
             },
-            mockRPC: this.mockDataExportRPCs,
-            session: {
-                ...this.mockSession,
-                get_file: function (params) {
-                    assert.step(params.url);
-                    params.complete();
-                },
+            mockRPC: function (route, args) {
+                if (route === '/web/dataset/call_kw/ir.async/search_read') {
+                    return Promise.resolve([]);
+                }
+                if (route === '/longpolling/poll') {
+                    self.pollPromise = testUtils.makeTestPromise();
+                    self.pollPromise.abort = () => {self.pollPromise.resolve([], $.Event())};
+                    return self.pollPromise;
+                }
+                if (route.startsWith('/web/async_export/')) {
+                    assert.step(route);
+                    let fakeJobId = -_.uniqueId();
```

Comment:
`-_.`, what's this??

Reply:
A negative unique id, sounded a smart way to ensure not any of those ID would map to a real job id but it may be over engineered

Reply:
It's important no having collision between jobs. I think if you extend notion of ID to string, you can do something like `_.uniqueId('fakeJob_')`, so that they have ID that looks visually '`fakeJob_<integer>`'. Don't have collision thanks to special prefix for these jobs and shape of id is more meaningful.